### PR TITLE
feat: create initial project template

### PR DESCRIPTION
**Pull request by yarkivaev**

### CODE REVIEW

**Comment by yarkivaev** [comment]
Go project
docker
postgresql

**Comment by ArtemSBulgakov** [comment]
You also should add License file

**Comment by evermake** [comment]
> You also should add License file

MIT?

**Comment by ArtemSBulgakov** [comment]
> MIT?

I think yes

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+.PHONY: lint
+lint: golangci-lint-installed  ## run linters
+	@golangci-lint run && echo "${CLR_GREEN}ok"
+
+
+# Helpers
+
+.PHONY: golangci-lint-installed
+golangci-lint:
```
Comment (evermake) [inline Makefile]: typo

**Review by evermake — APPROVED** [review]
LGTM

**Pull request merged**